Composer/packages/electron-server/src/appUpdater.ts

```diff
@@ -57,6 +57,7 @@ export class AppUpdater extends EventEmitter {
 
   public downloadUpdate() {
     if (!this.downloadingUpdate) {
+      this.downloadingUpdate = true;
```

Comment:
do we need to set this to false when it's finished?

Reply:
It already is being set to false in [`resetToIdle()`](https://github.com/microsoft/BotFramework-Composer/blob/master/Composer/packages/electron-server/src/appUpdater.ts#L120) which is called on error, finished downloading, or update unavailable.